mne_bids/tests/test_write.py

```diff
@@ -1400,16 +1401,16 @@ def test_write_raw_pathlike():
     raw = _read_raw_fif(raw_fname)
 
     bids_root = Path(_TempDir())
-    events_fname = (Path(data_path) / 'MEG' / 'sample' /
-                    'sample_audvis_trunc_raw-eve.fif')
+    events_fname = \
+        Path(data_path) / 'MEG' / 'sample' / 'sample_audvis_trunc_raw-eve.fif'
```

Comment:
Uff, why the backslash :) Wasn't it prettier before?

Reply:
No one likes the change to this huh? :) It failed by pylinter because of a line break after a binary operator. I thought pep8 was getting rid of those so we should change it

Reply:
Nope, ASAIK this is valid PEP8. Also the operator is overloaded here anyway, sooooo... ;)